logging.mm

```diff
@@ -1,6 +1,8 @@
 #include "logging.h"
 
 #include <objc/runtime.h>
+#include <CoreGraphics/CGAffineTransform.h>
```

Comment:
Nit: These should probably be `#import <CoreGraphics/CoreGraphics.h>` and `#import <UIKit/UIKit.h>`

Reply:
Fixed!